react/src/component/Dashboard.jsx

```diff
@@ -194,9 +194,9 @@ export default function Dashboard() {
                             </ListItem>
                             <ListItem component={Link} to={'/manage'} button>
                                 <ListItemIcon>
-                                    <span className="material-icons-outlined">groups</span>
+                                    <span className="material-icons-outlined">table_view</span>
```

Comment:
I think the icon table view is pretty good, but maybe we can find other icons that fit better.
![image](https://user-images.githubusercontent.com/88028886/220537335-0c093901-a272-4c17-b3d9-4eb0260e1d15.png)
![image](https://user-images.githubusercontent.com/88028886/220537409-98d15ded-fc24-4821-99d0-196538f80b9a.png)